README.md

```diff
@@ -42,6 +44,57 @@ use_frameworks!
 
 Then run `pod install` with CocoaPods 0.36 or newer.
 
+## Adding Runes as a Framework Dependency ##
+
+Runes can also be used as a low-overhead dependency for defining these
+operators for your own types. After linking to `Runes.swift`, you will be able
+to define implementations for your types, and your users will be able to
+import your framework to use these operators without needing to link against
+Runes directly.
```

Comment:
aslo throw in the line about how this allows your users to ignore Runes if they don't want it